### ISSUE TITLE
FR: (optional) on `subscribe` pass additional customer information to Stripe

### ISSUE THREAD

**Issue by jevon** [tierrun/tier#154]
### What are you trying to do?

I am attempting to search for a customer within the Stripe dashboard. 

I have:
- The customer's name
- the customer's email address

I cannot search on either of these fields because the only customer information available in Stripe is the Org ID. 

### How should we solve this?

When `tier subscribe` creates a *new* customer in Stripe, it should pass on the email address and name of the customer, if provided. 

The *relevant* optional fields which are accepted by Stripe are listed below. [A full list of fields is available here](https://stripe.com/docs/api/customers/create)

|  Name | Description  | 
|---|---|
| description  | An arbitrary string that you can attach to a customer object. It is displayed alongside the customer in the dashboard.  |
| email  | Customer’s email address. It’s displayed alongside the customer in your dashboard and can be useful for searching and tracking. This may be up to 512 characters.  |
| metadata  |  Set of [key-value pairs](https://stripe.com/docs/api/metadata) that you can attach to an object. This can be useful for storing additional information about the object in a structured format. |
| name  | The customer’s full name or business name. |
|  phone | The customer’s phone number.  |

### What is the impact of not solving this?

I would need to cross reference with Tier data. Likely by using `whois`

**Comment by bmizerany** [comment]
Agreed. What needs some thought is the feel for the CLI and semantics for subsequent calls to subscribe without customer information, and what an update looks like if no plan change is necessary.

My initial thoughts are:

**CLI**

Email set on first sighting of `org:acme` and the org is scheduled immediately for the free plan.

```
$ tier subscribe --email a@acme.com org:acme plan:free@0
```

Customer email changes, no plans or features will mean no changes to the subscription, and updates to the customer info are only required.

```
$ tier subscribe --email cfo@acme.com org:acme
```

Our customer wants to upgrade, and also changed their email in our upgrade form:

```
$ tier subscribe --email a@acme.com org:acme plan:free@0
```

I intentionally left out the other parameters because the CLI isn't the place for long-form data entry. That should be done in a WSYWIG or through the API, which is where I suspect 99.99% of customer information will be gathered and entered.

I'll spike out an implementation for this.

**Comment by bmizerany** [comment]
Fixed in #179
